src/libs/actions/Report.ts

```diff
@@ -3616,10 +3626,15 @@ function prepareOnboardingOptimisticData(
 
     // Guides are assigned and tasks are posted in the #admins room for the MANAGE_TEAM onboarding action, except for emails that have a '+'.
     const shouldPostTasksInAdminsRoom = engagementChoice === CONST.ONBOARDING_CHOICES.MANAGE_TEAM && !currentUserEmail?.includes('+');
-    const integrationName = userReportedIntegration ? CONST.ONBOARDING_ACCOUNTING_MAPPING[userReportedIntegration] : '';
     const adminsChatReport = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${adminsChatReportID}`];
     const targetChatReport = shouldPostTasksInAdminsRoom ? adminsChatReport : getChatByParticipants([CONST.ACCOUNT_ID.CONCIERGE, currentUserAccountID]);
-    const {reportID: targetChatReportID = '', policyID: targetChatPolicyID = ''} = targetChatReport ?? {};
+    const {reportID: targetChatReportID, policyID: targetChatPolicyID} = targetChatReport ?? {};
+
+    if (!targetChatReportID) {
+        return null;
```

Comment:
```suggestion
        return;
```

BTW why not just return `undefined`? Or make it `return undefined;`? Any reason to choose `null`?

Reply:
I just like returning null 😅 But I can apply your suggestion if you prefer!